gsl_core/templatetags/tests/test_gsl_filters.py

```diff
@@ -47,25 +47,65 @@ def test_remove_first_word():
     assert remove_first_word(mapping[PROJET_STATUS_PROCESSING]) == "En traitement"
 
 
-def test_create_alert_data():
-    assert create_alert_data(None, "Test description") == {
-        "is_collapsible": True,
-        "title": "Test description",
-    }
-    assert create_alert_data("valid", "Test description") == {
-        "is_collapsible": True,
-        "description": "Test description",
-        "title": "Projet accepté",
-    }
-    assert create_alert_data("cancelled", "Test description") == {
-        "is_collapsible": True,
-        "description": "Test description",
-        "title": "Projet refusé",
-    }
-    assert create_alert_data("other", "Test description") == {
-        "is_collapsible": True,
-        "description": "Test description",
-    }
+@pytest.mark.parametrize(
+    "extra_tags, title_expected, has_message, type_expected",
+    (
+        ("valid", "Projet accepté", True, None),
+        ("cancelled", "Projet refusé", True, None),
+        (
+            "provisionally_accepted",
+            "Projet accepté provisoirement",
+            True,
+            None,
+        ),
+        (
+            "provisionally_refused",
+            "Projet refusé provisoirement",
+            True,
+            None,
+        ),
+        (
+            "dismissed",
+            "Projet classé sans suite",
+            True,
+            None,
+        ),
+        (
+            "draft",
+            "Projet en traitement",
+            True,
+            None,
+        ),
+        ("info", None, "Test description", "info"),
+        ("alert", None, "Test description", "alert"),
+        ("other", None, "Test description", None),
+        (None, None, "Test description", None),
+    ),
+)
+def test_create_alert_data(extra_tags, title_expected, has_message, type_expected):
+    message_text = "Test description"
+    message = type("Message", (object,), {})()
+    message.__setattr__("message", message_text)
+    message.__setattr__("extra_tags", extra_tags)
```

Comment:
Ça ne marchait pas comme ça ? 

```suggestion
    message.message = message_text
    message.extra_tags = extra_tags
```

Reply:
ah si ! Mieux !